Assets/Scripts/Blocks/Block.cs

```diff
@@ -20,6 +20,8 @@ public abstract class Block : MonoBehaviour
     protected float currentLifeTime;
     protected bool isTouchingPlayer = false;
     [SerializeField] private GameObject destructionAnimation;
+    [SerializeField] public Texture destruction_texture;
+    private Texture recoveryTexture;
```

Comment:
Is there are reason why one is SerializeField and the other not, and also the variables should all me named with the lower-upper-case convention so destructionTexture instead of destruction_texture.